nsv13/code/__HELPERS/overmap.dm

```diff
@@ -12,12 +12,10 @@ Helper method to get what ship an observer belongs to for stuff like parallax.
 */
 
 /mob/proc/find_overmap()
-	var/obj/structure/overmap/OM = loc?.get_overmap() //Accounts for things like fighters and for being in nullspace because having no loc is bad.
-	if(!OM) //We're on the overmap Z-level itself, thus we don't belong to any ship
-		if(last_overmap)
-			last_overmap.mobs_in_ship -= src
+	var/obj/structure/overmap/OM = loc.get_overmap() //Accounts for things like fighters and for being in nullspace because having no loc is bad.
```

Comment:
Can loc ever be null here? I know it's uncommon but...

Reply:
if loc is null, something must have already broke catastrophically. It's not worth the (admittedly small) performance dent to check if it exists every time a mob moves